go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/indeedeng/iwf
 
-go 1.22
+go 1.23.0
```

Comment:
Looks like it's required by temporal cloud sdk: https://github.com/temporalio/cloud-sdk-go/blob/main/go.mod

Reply:
Could we just do 1.23? So that we're targeting all of the 1.23 go releases instead of 1.23.0 only?